.github/ranger.yml

```diff
@@ -26,7 +24,7 @@ labels:
     delay: 5s
     message: "Thanks for making your first contribution! :slightly_smiling_face:"
   extension-request:
-    action: close-faster
+    action: close
```

Comment:
@jsjoeio Ah that's too bad that there's no schema validator for these, I would expect some sort of warning if fields are unknown

Reply:
Right? Well based on what I read and saw in the config, I just assumed I could create my own custom actions like this: https://github.com/cdr/code-server/pull/3065/files#diff-15592ca299f8c76c553cc34e528b303896afa722a32d8f983cdcf67dfa7b38f1L11-L12

But turns out that is not the case 😂